src/HDWallet.cpp

```diff
@@ -60,6 +62,19 @@ PrivateKey HDWallet::getKey(TWPurpose purpose, TWCoinType coin, uint32_t account
     return PrivateKey(data);
 }
 
+Data HDWallet::getAionKey(TWPurpose purpose, TWCoinType coin, uint32_t account, uint32_t change, uint32_t address) const {
```

Comment:
Let’s call it getEd25519Key, other blockchains also use this curve

Reply:
We can also include the curve as a parameter.

Reply:
Yes, it’s an approach trezor-crypto uses, since only 2, 3 curves, I would like to differentiate them from method name.

Both works for me

Reply:
`getPublicKey(curve: Curve, ....)` might be easier to have it explicit